mailbox/event/event-rabbitmq/src/main/java/org/apache/james/mailbox/events/KeyRegistrationHandler.java

```diff
@@ -75,10 +76,10 @@
     }
 
     void start() {
-        sender.declareQueue(QueueSpecification.queue(eventBusId.asString())
+        sender.declareQueue(QueueSpecification.queue(EVENTBUS_QUEUE_NAME_PREFIX + eventBusId.asString())
             .durable(DURABLE)
             .exclusive(!EXCLUSIVE)
-            .autoDelete(!AUTO_DELETE)
+            .autoDelete(AUTO_DELETE)
```

Comment:
I'm ok with that as long as it can recover from outage.

There is a test about recovery after network failure in the Event bus contract. It was failing hence we turned that off.